daemon/daemon.cc

```diff
@@ -103,6 +103,7 @@ static auto constexpr Options = std::array<tr_option, 43>{
       { 941, "incomplete-dir", "Where to store new torrents until they're complete", nullptr, true, "<directory>" },
       { 942, "no-incomplete-dir", "Don't store incomplete torrents in a different location", nullptr, false, nullptr },
       { 'd', "dump-settings", "Dump the settings and exit", "d", false, nullptr },
+      { 943, "default-trackers", "Default trackers to be automatically added to public torrents", nullptr, true, "<list>" },
```

Comment:
_(nit)_ Update the description here slightly to match the non-sticky implementation

```suggestion
      { 943, "default-trackers", "Trackers for public torrents to use automatically", nullptr, true, "<list>" },
```